sql/core/src/test/resources/sql-tests/inputs/pgSQL/date.sql

```diff
@@ -228,9 +228,9 @@ SELECT f1 - date '2000-01-01' AS `Days From 2K` FROM DATE_TBL;
 -- test extract!
 --
 -- epoch
---
--- SELECT EXTRACT(EPOCH FROM DATE        '1970-01-01');     --  0
--- SELECT EXTRACT(EPOCH FROM TIMESTAMP   '1970-01-01');     --  0
```

Comment:
Could you keep the original form as much as possible by keeping `     --  0`?

Reply:
I tried that but the test failed here though on my laptop it passed. I removed it, just in case.

Reply:
Please, take a look at https://github.com/apache/spark/pull/25388#issuecomment-519469868

Reply:
Seems fixed as of https://github.com/apache/spark/pull/25357 . Does this still fail?

Reply:
I can revert the comments back and check if it is matter.

Reply:
Thanks, @MaxGekk .